frontend/dockerfile/dockerfile_test.go

```diff
@@ -5265,7 +5268,8 @@ RUN echo hello
 }
 
 func testNamedImageContextTimestamps(t *testing.T, sb integration.Sandbox) {
-	integration.SkipIfDockerd(t, sb, "direct push")
+	integration.SkipIfDockerdMoby(t, sb, "direct push", "named contexts timestamps")
+	integration.SkipIfDockerdSnapshotter(t, sb, "named contexts timestamps")
```

Comment:
https://github.com/crazy-max/buildkit/actions/runs/3257051098/jobs/5348086904#step:8:1232

```
=== CONT  TestIntegration/TestNamedImageContextTimestamps/worker=dockerd-containerd/frontend=builtin
    dockerfile_test.go:5349: 
        	Error Trace:	dockerfile_test.go:5349
        	            				run.go:86
        	            				run.go:189
        	Error:      	Should not be: time.Date(2022, time.October, 15, 20, 48, 54, 585592478, time.UTC)
        	Test:       	TestIntegration/TestNamedImageContextTimestamps/worker=dockerd-containerd/frontend=builtin
```

[TestNamedImageContextTimestamps.log](https://github.com/moby/buildkit/files/9793238/TestNamedImageContextTimestamps.log)

Reply:
I don't understand the connection in here. Assuming `DockerdSnapshotter` uses the image exporter directly and not moby.

Reply:
This means for `DockerdMoby` we have to skip because `direct push` and `named contexts timestamps` is not working but with `DockerdSnapshotter` only `named contexts timestamps` is not working: https://github.com/moby/buildkit/blob/c717d6aa7543d4b83395e0552ef2eb311f563aab/frontend/dockerfile/dockerfile_test.go#L5349